2024-11-03 17:07:25,946 - app.router.User.register - INFO - Attempting to register user with email: user@example1.com
2024-11-03 17:07:25,971 - passlib.handlers.bcrypt - WARNING - (trapped) error reading bcrypt version
Traceback (most recent call last):
  File "/home/cuong/Desktop/best_code/F5/backend/env/lib/python3.12/site-packages/passlib/handlers/bcrypt.py", line 620, in _load_backend_mixin
    version = _bcrypt.__about__.__version__
              ^^^^^^^^^^^^^^^^^
AttributeError: module 'bcrypt' has no attribute '__about__'
2024-11-03 17:07:26,341 - app.router.User.register - INFO - User registered successfully with email: user@example1.com, ID: 1
2024-11-03 17:07:29,200 - app.router.User.register - INFO - Attempting to register user with email: user@example1.com
2024-11-03 17:07:29,214 - app.router.User.register - WARNING - Registration failed: Email user@example1.com is already registered.
2024-11-03 17:08:13,444 - app.router.User.register - INFO - Attempting to register user with email: user1@example.com
2024-11-03 17:08:13,790 - app.router.User.register - INFO - User registered successfully with email: user1@example.com, ID: 2
2024-11-03 17:09:10,385 - app.router.User.auth - INFO - Received login attempt for username: user1@example.com
2024-11-03 17:09:10,749 - app.router.User.auth - INFO - Login successful for username: user1@example.com
2024-11-03 17:09:12,051 - app.router.User.auth - INFO - Received login attempt for username: user1@example.com
2024-11-03 17:09:12,391 - app.router.User.auth - INFO - Login successful for username: user1@example.com
2024-11-03 17:09:14,472 - app.router.User.auth - INFO - Received login attempt for username: user1@example.com
2024-11-03 17:09:14,787 - app.router.User.auth - INFO - Login successful for username: user1@example.com
2024-11-03 17:09:17,032 - app.router.User.auth - INFO - Received login attempt for username: user1@example.com
2024-11-03 17:09:17,346 - app.router.User.auth - INFO - Login successful for username: user1@example.com
2024-11-03 17:09:27,090 - app.router.User.auth - INFO - Received login attempt for username: user1@example.com
2024-11-03 17:09:27,410 - app.router.User.auth - INFO - Login successful for username: user1@example.com
2024-11-03 17:09:32,643 - app.router.User.auth - INFO - Received login attempt for username: user1@example.com
2024-11-03 17:09:32,961 - app.router.User.auth - INFO - Login successful for username: user1@example.com
2024-11-03 17:09:34,317 - app.router.User.auth - INFO - Received login attempt for username: user1@example.com
2024-11-03 17:09:34,635 - app.router.User.auth - INFO - Login successful for username: user1@example.com
2024-11-03 17:13:29,527 - app.router.User.auth - INFO - Received login attempt for username: user1@example.com
2024-11-03 17:13:29,910 - app.router.User.auth - INFO - Login successful for username: user1@example.com
2024-11-03 17:27:56,573 - app.router.User.register - INFO - Attempting to register user with email: cuong@gmail.com
2024-11-03 17:27:57,051 - app.router.User.register - INFO - User registered successfully with email: cuong@gmail.com, ID: 3
2024-11-03 17:29:29,562 - app.router.User.auth - INFO - Received login attempt for username: user1@example.com
2024-11-03 17:29:29,948 - app.router.User.auth - INFO - Login successful for username: user1@example.com
2024-11-03 17:29:47,131 - app.router.User.auth - INFO - Received login attempt for username: user1@example.com
2024-11-03 17:29:47,485 - app.router.User.auth - INFO - Login successful for username: user1@example.com
2024-11-03 18:29:17,806 - app.router.User.auth - INFO - Received login attempt for username: user1@example.com
2024-11-03 18:29:20,676 - app.router.User.auth - INFO - Received login attempt for username: user1@example.com
2024-11-03 18:29:21,157 - app.router.User.auth - WARNING - Login failed for username: user1@example.com - Incorrect password
2024-11-03 18:29:30,855 - app.router.User.auth - INFO - Received login attempt for username: user1@example.com
2024-11-03 18:29:31,178 - app.router.User.auth - WARNING - Login failed for username: user1@example.com - Incorrect password
2024-11-03 18:29:33,215 - app.router.User.auth - INFO - Received login attempt for username: user1@example.com
2024-11-03 18:29:33,534 - app.router.User.auth - WARNING - Login failed for username: user1@example.com - Incorrect password
2024-11-03 18:29:34,062 - app.router.User.auth - INFO - Received login attempt for username: user1@example.com
2024-11-03 18:29:34,247 - app.router.User.auth - INFO - Received login attempt for username: user1@example.com
2024-11-03 18:29:34,407 - app.router.User.auth - WARNING - Login failed for username: user1@example.com - Incorrect password
2024-11-03 18:29:34,722 - app.router.User.auth - WARNING - Login failed for username: user1@example.com - Incorrect password
2024-11-03 18:29:50,926 - app.router.User.auth - INFO - Received login attempt for username: user1@example.com
2024-11-03 18:29:51,289 - app.router.User.auth - INFO - Login successful for username: user1@example.com
2024-11-04 12:58:05,105 - app.router.User.register - INFO - Attempting to register user with email: admin@example.com
2024-11-04 12:58:05,167 - passlib.handlers.bcrypt - WARNING - (trapped) error reading bcrypt version
Traceback (most recent call last):
  File "/home/cuong/Desktop/best_code/F5/backend/env/lib/python3.12/site-packages/passlib/handlers/bcrypt.py", line 620, in _load_backend_mixin
    version = _bcrypt.__about__.__version__
              ^^^^^^^^^^^^^^^^^
AttributeError: module 'bcrypt' has no attribute '__about__'
2024-11-04 12:58:05,567 - app.router.User.register - INFO - User registered successfully with email: admin@example.com, ID: 4
2024-11-04 12:58:56,906 - app.router.User.auth - INFO - Received login attempt for username: admin@example.com
2024-11-04 12:58:57,276 - app.router.User.auth - INFO - Login successful for username: admin@example.com
2024-11-05 13:25:20,692 - app.router.User.register - INFO - Attempting to register user with email: user2@example.com
2024-11-05 13:25:20,942 - passlib.handlers.bcrypt - WARNING - (trapped) error reading bcrypt version
Traceback (most recent call last):
  File "/home/cuong/Desktop/best_code/F5/backend/env/lib/python3.12/site-packages/passlib/handlers/bcrypt.py", line 620, in _load_backend_mixin
    version = _bcrypt.__about__.__version__
              ^^^^^^^^^^^^^^^^^
AttributeError: module 'bcrypt' has no attribute '__about__'
2024-11-05 13:25:21,468 - app.router.User.register - INFO - User registered successfully with email: user2@example.com, ID: 5
2024-11-05 13:26:32,258 - app.router.User.auth - INFO - Received login attempt for username: user2@example.com
2024-11-05 13:26:32,647 - app.router.User.auth - INFO - Login successful for username: user2@example.com
2024-11-05 13:39:54,420 - app.router.User.auth - INFO - Received login attempt for username: user1@example.com
2024-11-05 13:39:54,791 - app.router.User.auth - INFO - Login successful for username: user1@example.com
2024-11-05 13:40:07,439 - app.router.User.auth - INFO - Received login attempt for username: user1@example.com
2024-11-05 13:40:07,774 - app.router.User.auth - INFO - Login successful for username: user1@example.com
2024-11-05 13:40:29,747 - app.router.User.auth - INFO - Received login attempt for username: user2@example.com
2024-11-05 13:40:30,093 - app.router.User.auth - INFO - Login successful for username: user2@example.com
2024-11-05 13:41:21,716 - app.router.User.register - INFO - Attempting to register user with email: user3@example.com
2024-11-05 13:41:22,097 - app.router.User.register - INFO - User registered successfully with email: user3@example.com, ID: 6
2024-11-05 13:41:36,636 - app.router.User.auth - INFO - Received login attempt for username: user3@example.com
2024-11-05 13:41:36,971 - app.router.User.auth - INFO - Login successful for username: user3@example.com
2024-11-05 13:43:08,054 - app.router.User.auth - INFO - Received login attempt for username: user1@example.com
2024-11-05 13:43:08,390 - app.router.User.auth - INFO - Login successful for username: user1@example.com
2024-11-05 13:45:23,392 - app.router.User.auth - INFO - Received login attempt for username: user1@example.com
2024-11-05 13:45:23,973 - app.router.User.auth - INFO - Login successful for username: user1@example.com
2024-11-05 13:56:37,235 - app.router.User.auth - INFO - Received login attempt for username: user1@example.com
2024-11-05 13:56:37,566 - app.router.User.auth - INFO - Login successful for username: user1@example.com
2024-11-05 14:00:08,119 - app.router.User.auth - INFO - Received login attempt for username: user1@example.com
2024-11-05 14:00:10,889 - app.router.User.auth - INFO - Received login attempt for username: user1@example.com
2024-11-05 14:00:17,468 - app.router.User.auth - INFO - Received login attempt for username: user1@example.com
2024-11-05 14:01:48,326 - app.router.User.auth - INFO - Received login attempt for username: user1@example.com
2024-11-05 14:01:49,622 - app.router.User.auth - INFO - Received login attempt for username: user1@example.com
2024-11-05 14:01:51,028 - app.router.User.auth - INFO - Received login attempt for username: user1@example.com
2024-11-05 14:01:52,621 - app.router.User.auth - INFO - Received login attempt for username: user1@example.com
2024-11-05 14:01:54,020 - app.router.User.auth - INFO - Received login attempt for username: user1@example.com
2024-11-05 14:01:58,207 - app.router.User.auth - INFO - Login successful for username: user1@example.com
2024-11-05 14:01:58,222 - app.router.User.auth - INFO - Login successful for username: user1@example.com
2024-11-05 14:01:58,240 - app.router.User.auth - INFO - Received login attempt for username: user1@example.com
2024-11-05 14:01:58,669 - app.router.User.auth - INFO - Login successful for username: user1@example.com
2024-11-05 14:01:58,750 - app.router.User.auth - INFO - Login successful for username: user1@example.com
2024-11-05 14:01:59,028 - app.router.User.auth - INFO - Login successful for username: user1@example.com
2024-11-05 14:02:00,587 - app.router.User.auth - INFO - Login successful for username: user1@example.com
2024-11-05 14:02:07,614 - app.router.User.auth - INFO - Received login attempt for username: user1@example.com
2024-11-05 14:02:07,988 - app.router.User.auth - INFO - Login successful for username: user1@example.com
2024-11-05 14:06:02,307 - app.router.User.auth - INFO - Received login attempt for username: user1@example.com
2024-11-05 14:06:02,762 - app.router.User.auth - INFO - Login successful for username: user1@example.com
2024-11-05 15:18:46,678 - app.router.User.auth - INFO - Received login attempt for username: admin@example.com
2024-11-05 15:18:46,762 - passlib.handlers.bcrypt - WARNING - (trapped) error reading bcrypt version
Traceback (most recent call last):
  File "/home/cuong/Desktop/best_code/F5/backend/env/lib/python3.12/site-packages/passlib/handlers/bcrypt.py", line 620, in _load_backend_mixin
    version = _bcrypt.__about__.__version__
              ^^^^^^^^^^^^^^^^^
AttributeError: module 'bcrypt' has no attribute '__about__'
2024-11-05 15:18:47,110 - app.router.User.auth - INFO - Login successful for username: admin@example.com
